server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestAddVotingConfigExclusionAction.java

```diff
@@ -80,7 +78,7 @@ static AddVotingConfigExclusionsRequest resolveVotingConfigExclusionsRequest(fin
         final var resolvedRequest = new AddVotingConfigExclusionsRequest(
             Strings.splitStringByCommaToArray(nodeIds),
             Strings.splitStringByCommaToArray(nodeNames),
-            request.paramAsTime("timeout", DEFAULT_TIMEOUT)
+            getAckTimeout(request)
```

Comment:
This is not a `AcknowledgedRequest`. But it happens to have the same default as ackTimeout. It also works similarly in that cluster state update will be applied even if it expires. So I think it's ok to reuse the same method here. But can we add a comment to say this is not really ackTimeout but reuse for convenience?

Reply:
Actually I just reverted that change, I think it's better just to leave `ackTimeout` for `AcknowledgedRequests`, principle of least surprise.